source/Engine.cpp

```diff
@@ -1588,6 +1588,10 @@ void Engine::HandleKeyboardInputs()
 {
 	Ship *flagship = player.Flagship();
 	
+	static const int landingWaitTime = 60;
```

Comment:
You are incorrect on the matter of constexpr.  From the Endless Sky coding standard, "constants should be defined as "static const" rather than using macros."  I have no objection to changing the coding standard, but my commit has to follow the standard until it is changed.

As for the location of landingWaitTime, we're both wrong.  According to the coding standard, "declare variables at the point of first use, not at the top of the function."  That means the two new constants both belong farther down in the function.